spec/models/movie_person_profile_spec.rb

```diff
@@ -92,88 +92,89 @@
     end
   end
 
-  describe '.parse_bio' do
+  describe 'private.parse_bio' do
```

Comment:
I chose to use `send` here in the specs instead of rewriting the public methods to account for these private methods. It's kind of a lazy move on my part. What's your preference for the codebase?